src/charts/utilization-bar/utilization-bar-chart.html

```diff
@@ -0,0 +1,34 @@
+<div class="pf-utilization-bar-chart">
+  <span ng-if="!layout || layout.type==='regular'">
+    <div ng-if="chartTitle" class="progress-description">{{chartTitle}}</div>
+    <div class="progress progress-label-top-right">
+      <div class="progress-bar" role="progressbar" ng-class="{'animate': animate,
+           'progress-bar-success': isOk, 'progress-bar-danger': isError, 'progress-bar-warning': isWarn}"
+           style="width:{{chartData.percentageUsed}}%" tooltip="{{chartData.percentageUsed}}% Used">
```

Comment:
I received the HTML/CSS from a RH-UXD designer and am required to match angular-patternfly to the base patternfly reflerence implementation HTML/CSS as close  as possible.